tensorly/regression/cp_plsr.py

```diff
@@ -86,6 +98,8 @@ def fit(self, X, Y):
         # Mean center the data, record info the object
         self.X_shape_ = T.shape(X)
         self.Y_shape_ = T.shape(Y)
+        self.original_X_ = T.copy(X)
```

Comment:
I'm not sure I'd want to store the full X and Y, it seems unnecessary and potentially very expensive (memory-wise): i'd let the user take care of that

Reply:
We can just calculate these numbers and save them in the object, but not save the `original_X` (only use them on the run). This is similar to how sklearn does.